manifest.webmanifest

```diff
@@ -0,0 +1,47 @@
+{
+  "lang": "en",
+  "dir": "ltr",
+  "name": "A-Blast",
+  "short_name": "A-Blast",
+  "description": "Save the world from the cutest creatures in the Universe!",
+  "start_url": "./",
+  "scope": "./",
+  "display": "fullscreen",
+  "theme_color": "#3d4d55",
+  "background_color": "#3d4d55",
+  "icons": [
+    {
+      "src": "assets/images/icon.png",
```

Comment:
FYI: these are losslessly compressed and resized icons. the original uncompressed icons live at `assets/images/`.